src/main/java/com/pd/algorithms/CountSorting.java

```diff
@@ -0,0 +1,69 @@
+package sorting;
+
+import java.util.Scanner;
+
+public class CountSorting {
+    static void countSort(int n, int... arr){
```

Comment:
Let's not make the method `static` and rather than printing the sorted array, please modify the method to return the sorted array.

Reply:
modified